src/components/snippets/FileManager.vue

```diff
@@ -200,8 +200,9 @@ export default Vue.extend({
       :draggable="dragEnabled"
       :root-location-disabled="rootLocationDisabled"
       :value="value"
-      :initial-items-per-page="initialItemsPerPage"
+      :items-per-page="itemsPerPage"
       :items-per-page-options="itemsPerPageOptions"
+      @update:itemsPerPage="$emit('update:itemsPerPage', $event)"
```

Comment:
does `:items-per-page.sync` work when you use pascalCase in the emitter?

![Screenshot from 2021-09-28 15-46-34](https://user-images.githubusercontent.com/4214172/135155748-22c35e9d-3be0-46a8-90f7-42aa89f5d0e4.png)

If so, I think this is fine to leave as-is.

Reply:
yeah in my testing I was using ke-bab case `:items-per-page.sync` inside of DIVE.  Should I unify them?  I just feel weird calling `$emit('update:this-test-variable')` and having `thisTestVariable` be non-searchable in the component file because it is in ke-bab case for the emit instead of camelCase.

Reply:
nah, it's cool like it is.